provisioner/powershell/provisioner.go

```diff
@@ -351,6 +354,13 @@ func (p *Provisioner) Provision(ctx context.Context, ui packersdk.Ui, comm packe
 		log.Printf("remote cleanup script failed to upload; skipping the removal of temporary files: %s; ", strings.Join(uploadedScripts, ","))
 	}
 
+	if p.config.PauseAfter != 0 { ui.Say(fmt.Sprintf("Pausing %s after this provisioner...", p.config.PauseAfter))
+		select {
+		case <-time.After(p.config.PauseAfter):
```

Comment:
I'm still pondering on this and maybe there's a good reason to keep your implementation, but `make ci-lint` does point out that this could be replaced by a plain `time.Sleep`.

I tend to agree with it, but feel free to correct me if there's something I miss.

Reply:
Converted, but I pulled this similar code from same [feature here on shell provisioner](https://github.com/hashicorp/packer/blob/main/provisioner/shell/provisioner.go#L370-L376), not sure how that is failing the ci check but my new code (the same thing) failed.

Reply:
Yes, I did notice that in other provisioners after reading yours; the CI should have pointed that out, my guess is that this was done before the CI-lint was in place, and it only checks in the new code for lint errors, hence why that other code doesn't trigger the CI pipeline.

We should probably uniformize that and make it compliant with the lint rule.

Thanks for the fix!